server/build.gradle

```diff
@@ -21,6 +21,7 @@ dependencies {
     // Armeria
     api libs.armeria
     api libs.armeria.thrift09
+    implementation project(path: ':server-jgit6')
```

Comment:
We can't depend on this for `implementation` because it makes a circular dependency as you noticed. Can we use `testImplementation`, guarded by an `if` instead? e.g.

```groovy
if (rootProject.ext.testJavaVersion >= 11) {
    testImplementation project(path: ':server-jgit6')
}
```

Reply:
thanks